tool/src/main/java/org/datacommons/tool/Processor.java

```diff
@@ -99,6 +105,10 @@ public static Integer process(Args args) throws IOException {
         processor.processNodes(Mcf.McfType.TEMPLATE_MCF);
       }
 
+      if (args.doStatChecks) {
+        processor.checkStats(graphAfterResolution);
```

Comment:
I wonder if its slightly more readable to fold the checkStats() logic in within the if (args.doResolution) above? So what happens in which case (e..g, the long comment in checkStats()) is obvious.  Feel free to disagree :)

```
if (args.doResolution) {
  ...

  graph = processor.resolveNodes();
  if (statChecker != null) {
    ... extract from graph ...
  }
} else {
  if (statChecker != null) {
     ... extract from nodesForVariousChecks ...
  }
}

...

// We've all along been adding stats to statChecker, do the final check.
if (statChecker != null) {
  statChecker.check();
}
```

Reply:
good suggestion, I feel like it makes things clearer. Updated